### PR TITLE
Cleans up multiline string assignments

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
This was produced with @N3X15 's excellent python script and a fair amount of manual fixing particularly around the clockcult blocks.

I'll be giving the changes another pass yet but reviews are still appreciated.

:cl: Naksu
code: multiline string assignments have been mostly concatenated into {" "} blocks, away from multiple append operations
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/air.dm [hunk 1]
```diff
@@ -44,22 +44,24 @@ SUBSYSTEM_DEF(air)
...
+		PN:[round(cost_pipenets,1)]|
+		AM:[round(cost_atmos_machinery,1)]
+		} 
+		AT:[active_turfs.len]|
+		EG:[excited_groups.len]|
+		HS:[hotspots.len]|
+		PN:[networks.len]|
+		HP:[high_pressure_delta.len]|
+		AS:[active_super_conductivity.len]|
+		AT/MS:[round((cost ? active_turfs.len/cost : 0),0.1)]"}
```
Comment (MrStonedOne) [inline code/controllers/subsystem/air.dm]: This isn't supposed to be multiline, did you confirm it shows up correctly in the mc tab

Comment (vuonojenmustaturska) [reply code/controllers/subsystem/air.dm]: It does not, and I haven't gone over every line manually but will be doing so over the weekend

Comment (MrStonedOne) [reply code/controllers/subsystem/air.dm]: These cases are best handled using a list and jointext/list.Join. keeps the string tree from having to allocate for the intermediate strings.

**Comment by vuonojenmustaturska** [comment]
That's a different but related thing, this is flattening multiple operations that only exist to provide newlines into one, the listification is for removing string concatenation and string table churn from the in-between steps

**Comment by SpaceManiac** [comment]
It seems like most of these only have newlines in the source code, not in the output. I would closely examine anything this touches that gets `to_chat`'d, as the additional newlines will actually become visible.

**Comment by vuonojenmustaturska** [comment]
let's do this in pieces instead

**Pull request closed without merging**